packages/editor/src/store/selectors.js

```diff
@@ -1098,20 +1097,10 @@ export const isPublishSidebarEnabled = createRegistrySelector(
 /**
  * Return the current block list.
  *
- * @param {Object} state
  * @return {Array} Block list.
  */
-export const getEditorBlocks = createSelector(
-	( state ) => {
-		return (
-			getEditedPostAttribute( state, 'blocks' ) ||
-			parse( getEditedPostContent( state ) )
-		);
-	},
-	( state ) => [
-		getEditedPostAttribute( state, 'blocks' ),
-		getEditedPostContent( state ),
-	]
+export const getEditorBlocks = createRegistrySelector(
+	( select ) => () => select( blockEditorStore ).getBlocks()
```

Comment:
To do: deprecate this selector.
Also maybe deprecated `getSuggestedPostFormat` and make a util with blocks as an arg.